sky/jobs/utils.py

```diff
@@ -918,6 +918,10 @@ def load_managed_job_queue(payload: str) -> List[Dict[str, Any]]:
     jobs = message_utils.decode_payload(payload)
     for job in jobs:
         job['status'] = managed_job_state.ManagedJobStatus(job['status'])
+        if 'user_hash' in job and job['user_hash'] is not None:
+            # Skip jobs that do not have user_hash info.
+            # TODO(cooperc): Remove check before 0.12.0.
+            job['user_name'] = global_user_state.get_user(job['user_hash']).name
```

Comment:
Just curious, if a user only runs `sky jobs launch` wihtout any `sky launch` before. Will the user_hash <-> user_name mapping be stored in the database?

Reply:
Should be added whenever any API server request is submitted. It's needed for `sky api status`, for instance.